public/app/features/transformers/editors/CalculateFieldTransformerEditor/BinaryOperationOptionsEditor.tsx

```diff
@@ -69,10 +121,10 @@ export const BinaryOperationOptionsEditor = (props: {
         <InlineField label="Operation" labelWidth={LABEL_WIDTH}>
           <Select
             allowCustomValue={true}
-            placeholder="Field or number"
+            placeholder="Field(s) or number"
             options={leftNames}
             className="min-width-18"
-            value={binary?.left}
+            value={JSON.stringify(binary?.left)}
```

Comment:
Is there a way to use `<Select/>` that can handle object values, instead of strings?